src/test/java/core/basesyntax/FruitShopServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax;
```

Comment:
The structure for the tests should be the same as the structure of the project with all packages(sevice, model, strategy....)

Reply:
Ok, I created packages for each test.